### ISSUE TITLE
Video player graph not working on latest (V3.5 build 3991) Linux AppImage

### ISSUE THREAD

**Issue by goalacj** [GoldenCheetah/GoldenCheetah#3394]
Hi, I've been trying the RLV training feature with the latest version of the Golden Cheetah Linux AppImage with no success on a Ubuntu 19.10 system.

There is no video image on Video Player graph.
The goldencheetah.log file shows this error when I click the Training tab:

[000000000cfe2b18] core libvlc error: No plugins found! Check your VLC installation.

Searching that error on the web I've tried setting this environment variable with no success:
export VLC_PLUGIN_PATH=/usr/lib/x86_64-linux-gnu/vlc/plugins

This error only occurs with the AppImage version of Golden Cheetah. If I launch the (outdated) version installed from the Ubuntu's repositories there is no problem and the video is played well.

**Comment by amtriathlon** [comment]
https://github.com/GoldenCheetah/GoldenCheetah/wiki/VLC-Troubleshooting

**Comment by goalacj** [comment]
Hi amtriathlon! thanks for your reply, I did forget to mention I found that wiki page searching for a solution and tried the proposed solution with no luck.

I executed the vlc-cache-gen command, although in Ubuntu 19.10 (VLC 3.0.8-2) the command should be:

sudo /usr/lib/x86_64-linux-gnu/vlc/vlc-cache-gen /usr/lib/x86_64-linux-gnu/vlc/plugins/ 

(it does not support the -f parameter either)

I've also just tried to create the symbolic link /usr/lib/vlc pointing to /usr/lib/x86_64-linux-gnu/vlc, and executing the vlc-cache-gen command.... but the result is the same: no video in Video Player graph and the same error in the goldencheetah.log when I click the Training tab:

[000000000cfe2b18] core libvlc error: No plugins found! Check your VLC installation.
